.github/integration/sda/aai-mock/ga4gh-broker.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS visas(
+    id              BIGINT AUTO_INCREMENT NOT NULL,
```

Comment:
Are there any special reasons to use `BIGINT` instead of `INT`? I don't think there will be billions of rows for the table.

Reply:
Don't ask me, all of that comes from the LS-AAI people.